xcode/Subconscious/Shared/AppTheme.swift

```diff
@@ -92,4 +92,52 @@ extension Color {
     static let fabTextPressed = fabText.opacity(0.5)
     static let fabTextDisabled = fabText.opacity(0.3)
     static let scrim = SwiftUI.Color(UIColor.tertiarySystemFill)
+    
+    // MARK: Brand colours
+    // Brand Mark
+    static let brandMarkPink = Color(uiColor: UIColor(red: 255/255, green: 163/255, blue: 186/255, alpha: 1)) // #FFA3BA
+    static let brandMarkViolet = Color(uiColor: UIColor(red: 197/255, green: 112/255, blue: 219/255, alpha: 1)) // #C570DB
+    static let brandMarkCyan = Color(uiColor: UIColor(red: 48/255, green: 255/255, blue: 233/255, alpha: 1)) // #30FFE9
+    static let brandMarkRed = Color(uiColor: UIColor(red: 255/255, green: 94/255, blue: 135/255, alpha: 1)) // #FF5E87
+    static let brandMarkPurple = Color(uiColor: UIColor(red: 132/255, green: 42/255, blue: 156/255, alpha: 1)) // #842A9C
```

Comment:
style: Should we simplify these to normalized values or leave in the 255 format?

Reply:
This works for me.

Non-blocking: we might could simplify with `SwiftUI.Color` constructors:

- https://developer.apple.com/documentation/swiftui/color/init(hue:saturation:brightness:opacity:)
- https://developer.apple.com/documentation/swiftui/color/init(_:red:green:blue:opacity:)

No need to change for this PR.